src/main/java/org/folio/circulation/resources/CheckOutByBarcodeResource.java

```diff
@@ -272,6 +276,11 @@ private CompletableFuture<Result<LoanAndRelatedRecords>> publishItemCheckedOutEv
 
     log.debug("publishItemCheckedOutEvent:: parameters records: {}", () -> records);
 
+    if (records.getLoan().isForUseAtLocation()) {
+      eventPublisher.publishUsageAtLocationEvent(
```

Comment:
PubSub is going to be deprecated and replaced with pure Kafka messaging. If you're adding new pubsub event types, please make sure they're mentioned in the pubsub deprecation plan https://folio-org.atlassian.net/wiki/spaces/DD/pages/1781531/PubSub+deprecation+plan